configure.ac

```diff
@@ -1152,6 +1160,45 @@ AS_IF([test "x$enable_arm64_crc32" = xyes], [
 	AC_CHECK_FUNCS([getauxval elf_aux_info sysctlbyname])
 ])
 
+# LoongArch Intrinsics define CRC32 functions in larchintrin.h.
+# These are supported by at least GCC and Clang.
+AC_MSG_CHECKING([if LoongArch CRC32 instruction is usable])
+AS_IF([test "x$enable_loongarch_crc32" = xno], [
+	AC_MSG_RESULT([no, --disable-loongarch-crc32 was used])
+], [
+	# Set -Werror here because -Wimplicit-function-declaration was
+	# only a warning in GCC <= 13. This does not need to be done
+	# with CMake because tests will attempt to link and the error
+	# will be reported then.
+	OLD_CFLAGS="$CFLAGS"
+	CFLAGS="$CFLAGS -Werror"
+
+	AC_COMPILE_IFELSE([AC_LANG_SOURCE([[
+#include <larchintrin.h>
+#include <stdint.h>
```

Comment:
`<stdint.h>` isn't needed in the feature test. It was used in the ARM64 feature test because `__crc32d()` parameters were expected to be `uint32_t` and `uint64_t` (and the return type is `uint32_t`).

Reply:
Done.